hydro_cli_examples/examples/topolotree/kruskals/default_gcp_regions_mst.py

```diff
@@ -0,0 +1,124 @@
+import folium
+import pandas as pd
+
+
+# Create the graph from the provided regions, latitudes, and longitudes
+regions = ["Iowa", "Oregon", "Northern Virginia", "South Carolina", "Belgium", "London", "Frankfurt",
+           "Singapore", "Taiwan", "Tokyo", "Sydney", "Sao Paulo", "Mumbai", "Montreal", "Netherlands",
+           "Los Angeles", "Salt Lake City", "Las Vegas", "Zurich", "Finland", "Jakarta", "Hong Kong",
+           "Osaka", "Seoul"]
+latitudes = [41.8780, 44.4268, 38.9072, 33.8361, 50.5039, 51.5074, 50.1109, 1.3521, 25.0330, 35.6895,
+             -33.8688, -23.5505, 19.0760, 45.5017, 52.3702, 34.0522, 40.7608, 36.1699, 47.3769, 60.1699,
+             -6.2088, 22.3193, 34.6937, 37.5665]
+longitudes = [-93.0977, -121.9707, -77.0369, -80.9450, 4.4699, -0.1278, 8.6821, 103.8198, 121.5654,
+              139.6917, 151.2093, -46.6333, 72.8777, -73.5673, 4.8952, -118.2437, -111.8910, -115.1398,
+              8.5417, 24.9384, 106.8456, 114.1694, 135.5022, 126.9779]
+
+
+class UnionFind:
+    def __init__(self, n):
+        self.parent = list(range(n))
+        self.rank = [0] * n
+
+    def find(self, x):
+        if self.parent[x] != x:
+            self.parent[x] = self.find(self.parent[x])
+        return self.parent[x]
+
+    def union(self, x, y):
+        root_x = self.find(x)
+        root_y = self.find(y)
+
+        if root_x != root_y:
+            if self.rank[root_x] < self.rank[root_y]:
+                self.parent[root_x] = root_y
+            elif self.rank[root_x] > self.rank[root_y]:
+                self.parent[root_y] = root_x
+            else:
+                self.parent[root_y] = root_x
+                self.rank[root_x] += 1
+
+
+def kruskals_algorithm(graph):
+    edges = []
+    for u in range(len(graph)):
+        for v in range(u + 1, len(graph)):
+            if graph[u][v] != 0:
```

Comment:
You should add some test cases that you know the correct minimum spanning tree of so that we can run kruskals and prims as well as any future MST algorithms we implement against those.